docs/src/adoc/index.adoc

```diff
@@ -7643,7 +7643,9 @@ https://jdbc.postgresql.org/documentation/head/binary-data.html[Pg-JDBC docs^]
 for upstream driver documentation.
 
 
-=== Spring 5
+=== Spring
+
+The `spring5` module supports Spring 5 and newer (Spring 6 has been lightly tested).
 
 This module integrates Jdbi and Spring 5 (or Spring Boot), including JTA support.
```

Comment:
Maybe update this line as well. AFAICS, we support Spring5/SpringBoot 2.x and Spring6/SpringBoot 6.x.